### PR TITLE
Update build config trigger display

### PR DESCRIPTION
**Pull request by spadgett**
Per @tlwu2013's mocks:

nodejs-ex details openshift origin 2018-07-14 09-34-36

/cc @rhamilto @tlwu2013

### CODE REVIEW

**Comment by spadgett** [comment]
@tlwu2013 It's pretty easy to switch between the options. I liked option 3 because it keeps the columns balanced when switching between the build config and build pages and gives more room for the webhooks table.

File: frontend/public/components/utils/triggers.tsx [hunk 1]
```diff
@@ -62,10 +73,8 @@ export const Triggers: React.SFC<TriggersProps> = ({ resource }) => {
...
 
-/* eslint-disable no-undef */
-export type TriggersProps = {
+export type WebhookTriggersProps = {
   resource: K8sResourceKind;
 };
-/* eslint-enable no-undef */
 
-Triggers.displayName = 'Triggers';
+WebhookTriggers.displayName = 'Triggers';
```
Comment (jhadvig) [inline frontend/public/components/utils/triggers.tsx]: Shouldn't this be `WebhookTriggers.displayName = 'WebhookTriggers'` ?

**Comment by jhadvig** [comment]
One small nit/question, otherwise looks good. Tested the change and :+1:

**Comment by spadgett** [comment]
Updated this to obscure the secret again for now.

/hold cancel

**Pull request merged**